src/io.jl

```diff
@@ -81,6 +74,17 @@ function create_parameters_and_sets_from_file(input_folder::AbstractString)
     return params, sets
 end
 
+"""
+    get_df(path, file_name, dataset_type)
+
+Reads the csv with file_name at location path, with 2 header rows. Then validates it using the dataset_type.
+"""
+function get_df(path, file_name, dataset_type)
```

Comment:
You could accept additional keyword arguments, that are then passed on to `CSV.read`.

- That would allow us to handle a much wider variety of CSVs trivially.  
- Then you can pass `header = 2` in the call to `get_df` making it explicit. This also limits all CSV related details (dir, file name, header rows, schema, etc) to the same part of the code.